src/main/java/com/gluonhq/NativeBaseMojo.java

```diff
@@ -270,7 +270,7 @@ private String getProjectClasspath() {
                 .collect(Collectors.toList());
     }
 
-    private Optional<String> getGraalvmHome() {
+    Optional<String> getGraalvmHome() {
         if (graalvmHome != null) {
```

Comment:
This looks  a little convoluted to me.. I think it can be simpler, something like:
`Optional.ofNullable(graalvmHome).or( ()-> Optional.ofNullable(System.getenv("GRAALVM_HOME"))`

Reply:
Fixed